### ISSUE TITLE
Updater not working

### ISSUE THREAD

**Issue by TonyMarston** [HeidiSQL/HeidiSQL#1713]
**Preconditions**

* HeidiSQL version: 12.2.0.6578 (64 bit)
* Database type and version: Mysql 8.0.31
* OS: Windows 10

**Describe the bug**

**To Reproduce**

**Screenshots or Crash reports**

**Comment by ansgarbecker** [comment]
Yes, that's a known issue, which I struggled with in the recent builds. I think I have already fixed it. But it would be good to have someone else testing this. Could you please do that:

* check with the Windows task manager if there are dead processes from `HeidiSQL_updater.exe` - please **terminate these**
* download build [6581](https://www.heidisql.com/builds/heidisql64.r6581.exe) (which has my bugfix for the updater) and **overwrite** `C:\Program Files\HeidiSQL\heidisql.exe` with it
* start that exe file
* use "check for updates" to see if the newer build will be downloaded and replaces the old one.

Thanks in advance!

**Comment by ansgarbecker** [comment]
Thank you for testing and quick feedback!
